readme.txt

```diff
@@ -16,27 +16,28 @@ Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable
 - FIDO Universal 2nd Factor (U2F)
 - Backup Codes
 - Dummy Method (only for testing purposes)
-
-For more history, see [this post](https://stephanis.info/2013/08/14/two-cents-on-two-factor/).
+
+For more history, see [this post](https://stephanis.info/2013/08/14/two-cents-on-two-factor/).
 
 
 == Screenshots ==
 
 1. Two-factor options under User Profile.
 2. U2F Security Keys section under User Profile.
+3. Two-factor Wordpress Login Screen

```

Comment:
Could we change this to describe the two-factor method shown in the screenshot and using the correct spelling of "WordPress". Maybe something like `3. Email code authentication method input during WordPress login`?